Post release RVS install update (#1275)
ROCm/ROCmValidationSuite: docs/install/installation.rst

Prerequisites
=============

Install the ROCm Core SDK before installing RVS.

> For instructions, see `Install AMD ROCm <https://rocm.docs.amd.com/>`__. Use the

Reviewer: Should be better if we cross-reference to the install docs: https://rocm.docs.amd.com/en/latest/install/rocm.html
```suggestion
For instructions, see :doc:`Install AMD ROCm <rocm:install/rocm>`. Use the
```

selector panel on that page to view instructions appropriate for your system
environment.

ROCm installation path
======================

The ROCm installation path depends on what installation method was selected during
ROCm installation. After installation, the ROCm Core SDK will be deployed to this

> location and contains all the core ROCm directories such as ``bin``, ``include``,

Reviewer: ```suggestion
path location and contains all the core ROCm directories such as ``bin``, ``include``,
```

> ``lib``, etc. When configuring the environment for RVS, you must set the

Reviewer: ```suggestion
and ``lib``. When configuring the environment for RVS, you must set the
```

> ``ROCM_INSTALL_PATH`` variable to the ROCm core installation directory

Reviewer: ```suggestion
``ROCM_INSTALL_PATH`` environment variable to the ROCm core installation directory
```

   The ROCm repositories must be set up before installing RVS. This repository
   setup is part of the ROCm Core SDK installation.

> Package manager uninstalling

Reviewer: ```suggestion
Package manager uninstallation
```

> =============================

Reviewer: ```suggestion
===================================
```

         .. tab-item:: User

            .. tab-set::

               .. tab-item:: Ubuntu
                  :sync: ubuntu

                  .. code-block:: bash

                     tee --append ~/.bashrc << EOF
                     export EXTRAS_PATH=$EXTRAS_INSTALL_PATH
                     export ROCM_PATH=$ROCM_INSTALL_PATH
                     export PATH=\$EXTRAS_PATH/bin:\$ROCM_PATH/bin:\$PATH
                     export LD_LIBRARY_PATH=\$EXTRAS_PATH/lib:\$ROCM_PATH/lib:\$ROCM_PATH/lib/llvm/lib:\$LD_LIBRARY_PATH
                     EOF

                     source ~/.bashrc

               .. tab-item:: RHEL
                  :sync: rhel

                  .. code-block:: bash

                     tee --append ~/.bashrc << EOF
                     export EXTRAS_PATH=/opt/rocm/extras-7
                     export ROCM_PATH=$ROCM_INSTALL_PATH
                     export PATH=\$EXTRAS_PATH/bin:\$ROCM_PATH/bin:\$PATH
                     export LD_LIBRARY_PATH=\$EXTRAS_PATH/lib:\$ROCM_PATH/lib:\$ROCM_PATH/lib/llvm/lib:\$LD_LIBRARY_PATH
                     EOF

                     source ~/.bashrc

5. Verify your installation.

   .. code-block:: bash

      rvs -g

> Tarball uninstalling

Reviewer: ```suggestion
Tarball uninstallation
```

> ====================

Reviewer: ```suggestion
=========================
```

1. Remove the installation directory.

   .. important::
      The following command assumes you're working with the
      ``EXTRAS_INSTALL_PATH`` directory set to ``/opt/rocm/extras-7``. If you
      chose a different directory name when installing RVS, adjust the command
      accordingly.

   .. code-block:: bash

      sudo rm -rf /opt/rocm/extras-7

> 2. Remove RVS environment configuration.

Reviewer: ```suggestion
2. Remove the RVS environment configuration.
```
